app/views/_macros/components.nunjucks

```diff
@@ -228,7 +228,7 @@
 {% endmacro %}
 
 {% macro panel(content) %}
-  <article class="panel">
+  <article class="panel reading-width">
```

Comment:
Should this be an `article`? Article is supposed to be self-contained redistributable piece of content. This looks like it heavily relies on context.